release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2720,6 +2720,78 @@ This section will continue to be updated over time to provide notes on enhanceme
+{product-title} release 4.15.12, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:2664[RHSA-2024:2664] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:2669[RHSA-2024:2669] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.12 --pullspecs
+----
+
+[id="ocp-4-15-12-enhancements"]
+==== Enhancements
+
+The following enhancements are included in this z-stream release:
+
+[id="ocp-4-15-12-apiversion-upgrade-v1beta1"]
+===== API Version for CRD ClusterTriggerBinding, TriggerTemplate, and EventListener upgraded from v1alpha1 to v1beta1
+
+* Previously, the API version for CRD ClusterTriggerBinding, TriggerTemplate, and EventListener was v1alpha1. With this release, the API version is upgraded to v1beta1 so that the pipelines plugin supports the latest Pipeline Trigger API version for CRD ClusterTriggerBinding, TriggerTemplate, and EventListener. (link:https://issues.redhat.com/browse/OCPBUGS-31445[*OCPBUGS-31445*])
+
+[id="ocp-4-15-12-pipeline-list-page-improvement"]
+===== PipelineRun list view performance improvement
+
+* Previously, in the PipelineRun list page, all of the TaskRuns were fetched and separated based on their PipelineRun name. With this release, TaskRuns are only fetched for failed and cancelled PipelineRuns, and a caching mechanism is added to fetch the PipelineRuns and TaskRuns that are associated with the failed and cancelled PipelineRuns. (link:https://issues.redhat.com/browse/OCPBUGS-31799[*OCPBUGS-31799*])
+
+[id="ocp-4-15-12-installer-handling-new-character"]
+===== Installer handles the escaping of the "%" character
+
+* Previously, if a cluster was installed using proxy and the proxy information contained escaped characters in the format "%XX", the installation failed. With this release, the Installer now handles the escaping of the "%"" character. (link:https://issues.redhat.com/browse/OCPBUGS-32259[*OCPBUGS-32259*])
+
+[id="ocp-4-15-12-cfe-evaluation-status"]
+===== Cluster Fleet Evaluation status information added to the Machine Config Operator
+
+* Previously, the Machine Config Operator (MCO) did not include the Cluster Fleet Evaluation (CFE) Status. With this release, the CFE status information is added to the MCO and available to customers. (link:https://issues.redhat.com/browse/OCPBUGS-32922[*OCPBUGS-32922*])
+
+[id="ocp-4-15-12-operatorhub-filter-rename-FIPS"]
+===== OperatorHub filter renamed from "FIPS Mode" to "Designed for FIPS"
+
+* Previously, OperatorHub included a filter named "FIPS Mode". With this release, that filter is named "Designed for FIPS". (link:https://issues.redhat.com/browse/OCPBUGS-32933[*OCPBUGS-32933*])
```

Comment:
Since these are UI labels, think they should be bold markup in the body text. But since we don't markup titles, tho... I guess just drop the quotes for some sort of consistency?

```suggestion
===== OperatorHub filter renamed from FIPS Mode to Designed for FIPS

* Previously, OperatorHub included a filter named *FIPS Mode*. With this release, that filter is named *Designed for FIPS*. (link:https://issues.redhat.com/browse/OCPBUGS-32933[*OCPBUGS-32933*])
```